arangod/RestHandler/RestShutdownHandler.cpp

```diff
@@ -68,6 +70,16 @@ RestStatus RestShutdownHandler::execute() {
     }
   }
 
+  auto const& softShutdownFeature{server().getFeature<SoftShutdownFeature>()};
+  auto& softShutdownTracker{softShutdownFeature.softShutdownTracker()};
+
+  if (_request->requestType() == rest::RequestType::GET) {
+    VPackBuilder builder;
+    softShutdownTracker.toVelocyPack(builder, softShutdownTracker.getStatus());
```

Comment:
Calling a method on the soft shutdown tracker and passing it its own internals looked a bit strange.
I figured that `toVelocyPack` is static, so it's how it is. But I would prefer a non-static method `toVelocyPack` on the tracker that dumps it own status into a result. (it may just fetch its own status and call the static method with 2 parameters).

Reply:
Fine, will add that, although we are not passing its internals, but a "public" result. Anyway, will do.

Reply:
Done.